doc/whats_new/v0.23.rst

```diff
@@ -259,6 +259,16 @@ Changelog
   of strictly inferior for maximum of `absgrad` and `tol` in `utils.optimize._newton_cg`.
   :pr:`16266` by :user:`Rushabh Vasani <rushabh-v>`.
 
+:mod:`sklearn.manifold`
+...........................
+
+- |Feature| Support of multidimensional scaling method,
+  which uses Singular Value Decomposition (SVD) in :class:`manifold.MDS`.
+  User can choose whether to use new SVD- or old SMACOF-based method
```

Comment:
I wouldn't use the terms new and old since they might suggest different meanings depending on when they are read

Reply:
Done